arch/arm/boot/dts/overlays/README

```diff
@@ -2061,6 +2061,12 @@ Params: krnbt                   Set to "on" to enable autoprobing of Bluetooth
                                 driver without need of hciattach/btattach
 
 
+Name:   i2c-mlx90640
```

Comment:
README entries must be in alphabetical order, so it should come before i2c-mux (like the Makefile entry).
However, I'm not sure that the `i2c-` adds anything here (there is no SPI option), so would you consider just calling it `mlx90640`?